appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/ProjectList.java

```diff
@@ -283,6 +283,9 @@ public int getSelectedProjectsCount() {
     }
   }
 
+  public boolean listContainsProjects() {
```

Comment:
Since the type is already `ProjectList`, the method name could just be `containsProjects`, although `hasProjects` might also be appropriate.

Reply:
> Since the type is already `ProjectList`, the method name could just be `containsProjects`, although `hasProjects` might also be appropriate.

I was intentionally not using the same name as the folder method, but upon consideration, that's probably not a concern.